### ISSUE TITLE
Routing doesn't work properly on elemental-ui.com

### ISSUE THREAD

**Issue by JedWatson** [elementalui/elemental#36]
Refreshing a url, e.g. http://elemental-ui.com/css on the live domain doesn't load the route correctly. It works in dev, so I'm not sure what's wrong. Needs investigation.

**Comment by JedWatson** [comment]
I've asked @LennyBoyatzis to take a look at this, suspect it's some setting on `gh-pages` that allows the `index.html` page to handle any route.

**Comment by NogsMPLS** [comment]
I believe this is fixed with the react 0.14/ react-router 1.0.0 changes I put in the other week, might need more thorough testing.

**Comment by JedWatson** [comment]
I did some experimenting last week, and spoke to GitHub, and it turns out the only way to avoid 404s is to _duplicate_ the `index.html` at _ALL_ the paths. The fix in place when react 0.14 was released was actually that experiment, I committed copies of the files to the `gh-pages` branch directly.

I've added the duplicates to the `site/src` directory for now, which solves the problem, it would be nice to automatically do this as part of the release process though rather than maintaining a bunch of duplicate `.html` files. I'll open another issue for that.
